### DISCUSSION TITLE
Compatible boards

### DISCUSSION THREAD

**Discussion by mingusdude** [FundamentalFrequency/.github#66] [Software]
So I'm trying to put together a list of compatible SBCs for the daw, and I could really use everyone's input. Obviously all the Raspberry flavored boards(4b,3b,cm4) will perform better out of the box, but has anyone had luck with any other boards? I have a Banana Pi M5 coming in the mail, that I will run a test on before pouring it to purpose in another project; and I'm going to try a Rockpi4 if it ever hits my mailbox, but I'd love to hear about everyone else's successes AND failures.  This project brings me real joy, and I want to spread that around!  Thanks guys

**Comment by insipx** [comment]
Any updates on this? Were you able to get it running with any alternatives boards? I'm interested since RaspberryPi is very difficult to pickup at MSRP atm. So I'd be enthralled if it works well with a different board. The rock would be especially interesting

**Comment by rocket-ron** [comment]
I’ve finally gotten around to trying the RockPi4 board. Initial results are not good as I’m not able to get the RockPi to work with the display very easily. Also LMN-3 DAW segfaults when running and doesn’t seem to find the Teensy 4.1 on the USB port. I’m going to take a step back and get things working on a RPi4 first then try again on the RockPi4.

**Reply by stonepreston** [reply]
ive seen people report segfaults when running as root, so make sure you are running the DAW as a non root user

**Comment by mingusdude** [comment]
Ohhh that's super good to know! I've been playing with a banana pi m5,
which spoiler alert,, is a little difficult. No WiFi out of the box, but
there is an "official" rpios image on bpi's website. So at least that's
something! Ill pay more once I'm off later and let everyone know if I can
make it work.

**Comment by benpocalypse** [comment]
I had been making a custom wooden enclosure for my LMN-3 that included 2 displays, a battery, built-in speakers, etc. but because you can't source any RPi 4's anymore I had been doing this with an RPi 3b I had laying around. Unfortunately the power budget exceeds what can be sourced via micro USB, so I had to shelve it for awhile.

Well, I've found the Banana Pi BPI CM5, and have ordered one. It looks like it should be relatively straight forward to get all the LMN-3 stuff up and running on it, so once I get it and mess around, I'll be sure to report back.
